samples/05.custom-components/d.reaction-buttons/README.md

```diff
@@ -123,19 +123,21 @@ The `sendPostBack` function will be retrieve from Web Chat hooks via `useSendPos
 + const BotActivityDecorator = ({ activityID, children }) => {
 +   const postActivity = usePostActivity();
 +
-+   const handleDownvoteButton = useCallback(() => {
-+     postActivity({
-+       type: 'messageReaction',
-+         reactionsAdded: [{ activityID, helpful: -1 }]
-+     });
-+   }, [activityID, postActivity]);
-+
-+   const handleUpvoteButton = useCallback(() => {
-+     postActivity({
-+       type: 'messageReaction',
-+       reactionsAdded: [{ activityID, helpful: 1 }]
-+     });
-+   }, [activityID, postActivity]);
++   const addMessageReaction = helpful => {
+            postActivity({
```

Comment:
I think these will need the `+` in the .md file to indicate they are additions to the code.